packages/scripts/src/dbt/generateModel.js

```diff
@@ -36,24 +36,26 @@ async function getSchemas(client) {
       WHERE schema_name not similar to 'pg%|information_schema|sync_snapshots|reporting'
       ORDER BY schema_name`,
     )
-  ).rows.map((table) => table.schema_name);
+  ).rows.map(table => table.schema_name);
 }
 
 /**
  * @param {pg.Client} client
  * @param {string} schemaName
- * @returns A list of table names that the given schema name has
+ * @returns {Array<{ name: string; oid: number }>} A list of table names that the given schema name has
  */
 async function getTablesInSchema(client, schemaName) {
   return (
     await client.query(
-      `SELECT DISTINCT table_name
+      `SELECT DISTINCT
+        table_name as name,
+         ('"' || table_schema || '"."' || table_name || '"')::regclass::oid as oid
```

Comment:
Oh this is so cool I had no idea you could get from the human readable table name to the `oid` this way!

Reply:
I didn't either but I knew you could do `'foo.bar'::regclass::oid` and thought... hmm, i wonder if it works dynamically... and it does! i love this stuff